kustomize/base/quay.deployment.yaml

```diff
@@ -16,9 +16,13 @@ spec:
     spec:
       serviceAccountName: quay-app
       volumes:
-        - name: configvolume
-          secret:
-            secretName: quay-config-secret
+        - name: config
```

Comment:
Switch to use k8s projected volume to mount both `Secrets`containing config bundle and TLS cert/key into the same `/conf/stack` directory in the container.